libs/npm/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "fine-uploader": "5.14.5"
```

Comment:
Currently there were two developers which announced their interest for the maintenance of the library. 
I currently rate the amount of other contributers and the active state of the project high enough...

Beside this: with one of the next commits this dependency will switch from npm to bower. But the "introduction" of bower will be part of another PR.